Dockerfile

```diff
@@ -3,7 +3,13 @@ WORKDIR /src
 ENV GO111MODULE=on
 RUN go get -u github.com/swaggo/swag/cmd/swag@v1.6.5
 COPY . /src
-RUN swag init && CGO_ENABLED=0 go build -o main
+ARG WHARF_GITHUB_VERSION="local docker"
+ARG WHARF_GITHUB_CI_GIT_COMMIT="HEAD"
+ARG WHARF_GITHUB_CI_BUILD_REF="0"
+RUN deploy/update-version.sh version.go \
+		&& swag init \
+		&& go get -t -d \
```

Comment:
this wasn't needed before, is it now?

Reply:
Bit of a copy-paste issue. I'll remove it.

Reply:
Received the following error without it:

```
go: downloading golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
go: downloading github.com/PuerkitoBio/purell v1.1.0
go: downloading github.com/go-openapi/jsonpointer v0.17.0
go: downloading github.com/go-playground/locales v0.13.0
go: downloading github.com/go-openapi/swag v0.17.0
go: downloading github.com/mailru/easyjson v0.0.0-20180823135443-60711f1a8329
docs/docs.go:12:2: missing go.sum entry for module providing package github.com/alecthomas/template (imported by github.com/iver-wharf/wharf-provider-github/docs); to add:
	go get github.com/iver-wharf/wharf-provider-github/docs
```